styles/main.css

```diff
@@ -129,20 +132,43 @@ input {
   margin-top: 25px;
 }
 .remove.icon:before {
-  content: '';
+  content: "";
   position: absolute;
   width: 15px;
   height: 1px;
   background-color: currentColor;
   -webkit-transform: rotate(45deg);
-          transform: rotate(45deg);
+  transform: rotate(45deg);
 }
 .remove.icon:after {
-  content: '';
+  content: "";
   position: absolute;
   width: 15px;
   height: 1px;
   background-color: currentColor;
   -webkit-transform: rotate(-45deg);
-          transform: rotate(-45deg);
-}
\ No newline at end of file
+  transform: rotate(-45deg);
+}
+
+.empty-list-container {
+}
+.empty-img {
+  width: 100%;
+  height: auto;
+  margin-top: 25px;
+}
```

Comment:
Can you follow the pattern of jumping one line?

Reply:
Done